src/api/rem-types.ts

```diff
@@ -0,0 +1,42 @@
+export enum Upgrade {
+  HeatPump = 'hvac__heat_pump_seer18_hspf10',
+  HeatPumpAndWeatherization = 'combination__hvac_seer18_hspf10__weatherization',
+  WaterHeater = 'water_heater__heat_pump_uef3.35',
+  Weatherization = 'weatherization__insulation_air_duct_sealing',
+}
+
+export enum HeatingFuel {
+  Electricity = 'electricity',
+  FuelOil = 'fuel_oil',
+  NaturalGas = 'natural_gas',
+  Propane = 'propane',
+}
+
+export enum WaterHeatingFuel {
+  Electricity = 'electricity',
+  FuelOil = 'fuel_oil',
+  NaturalGas = 'natural_gas',
+  Propane = 'propane',
+}
```

Comment:
These are defined separately in the API, so I'm reflecting that here, although I don't expect them to ever have different contents.